app/javascript/packs/vue/repository_search.js

```diff
@@ -5,6 +5,36 @@ import RepositorySearchContainer from '../../vue/repository_search/container.vue
 Vue.use(TurbolinksAdapter);
 Vue.prototype.i18n = window.I18n;
 
+// register click-ouside directive to detect when inputs loss focus
+let handleOutsideClick;
```

Comment:
As this looks generally useful, we should extract it into a separate file (I suggest javascript/vue/directives/click-outside.js), and import it here.